core/core-workflow-steps/src/main/java/org/eclipse/sw360/antenna/csvreader/CSVReader.java

```diff
@@ -255,15 +255,19 @@ private void addOptionalArtifactFacts(CSVRecord record, Artifact artifact) {
         }
         if (checkIfRecordIsMappedAndNotEmptyForParameter(record, PATH_NAME)) {
             String pathName = record.get(PATH_NAME);
-            String absolutePathName = makePathAbsolute(pathName);
-            artifact.addFact(new ArtifactPathnames(absolutePathName));
+            Path path = getPathFromPathname(pathName);
+            if (path.toFile().exists()) {
```

Comment:
You could use java.nio.file.Files.exists(path) to check whether the path exists without having it to convert to a File.